src/utilities/currency/index.ts

```diff
@@ -0,0 +1,10 @@
+export function format(value: number, precision: number = 2): string {
+  return value
+    .toFixed(precision)
+    .replace('.', ',')
+    .replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1.');
+}
+
+export function parse(value: string = ''): number {
```

Comment:
I don't know if we need to specify a default value for `value` arg...

Reply:
without the default value, when we cast the `parse()` function without any parameter TS will throw an error

"An argument for 'value' was not provided."

I think that's ok, I just need to remove one test case

Reply:
In this function `value` is `required`, so I think that it's ok to throw an error in that case.